src/main/java/com/renatusnetwork/momentum/data/stats/StatsDB.java

```diff
@@ -381,6 +382,17 @@ public static boolean isPlayerInDatabase(String playerName)
         return !playerResult.isEmpty();
     }
 
+    public static String getUUIDByName(String playerName) {
+        Map<String, String> playerResult = DatabaseQueries.getResult(
+                DatabaseManager.PLAYERS_TABLE,
+                "uuid",
+                " WHERE name=?", playerName
+        );
+        if (playerResult.isEmpty()) return null;
```

Comment:
you should just do:

return playerResult.get("uuid");

and no if is Empty or keySet


the DatabaseQueries#getResult will return a map of the column name it found to the value associated with it. so if column name uuid is found (which you are selecting for), it will return the players uuid from the db, otherwise it will just be null which is what we want.

Reply:
oh i didnt know that
fixed